tests/vxlan/test_vnet_vxlan.py

```diff
@@ -0,0 +1,167 @@
+import json
+import logging
+import pytest
+import vnet_config as vc
+import yaml
+
+from datetime import datetime
+from jinja2 import Template
+from netaddr import IPAddress
+from os import path
+from time import sleep
+from tests.ptf_runner import ptf_runner
+from vnet_utils import *
```

Comment:
I'm surprised that LGTM didn't call this out but it's generally not a good practice to `import *`.